generators/app/prompts.js

```diff
@@ -106,18 +111,31 @@ function askForBigBangOperations(context, done) {
             name: 'components',
             message: 'Which Kafka components would you like to generate?',
             choices: componentChoices(),
-            default: []
+            default: [],
+            validate: input => (_.isEmpty(input) ? 'You have to choose at least one component' : true)
         },
         {
-            when: response => response.components.includes('consumer') || response.components.includes('producer'),
+            when: response =>
+                response.components.includes(constants.CONSUMER_COMPONENT) || response.components.includes(constants.PRODUCER_COMPONENT),
             type: 'checkbox',
             name: 'entities',
             message: 'For which entity (class name)?',
             choices: entitiesChoices(context),
-            default: []
+            default: [],
+            validate: input => (_.isEmpty(input) ? 'You have to choose at least one option' : true)
+        },
+        {
+            when: response => response.entities.includes(constants.NO_ENTITY),
```

Comment:
in Big bang mode i can generate components for only one no_entity components, is it normal?
![57-PR-review-bbang](https://user-images.githubusercontent.com/44546269/83668632-93392d80-a5d0-11ea-9db6-8dbff66154b3.png)

Reply:
Yes because you're not looping like in incremental mode.

Reply:
Let's make it this way for the moment and see if there is some needs in the future!